backend/apps/ecommerce/tests.py

```diff
@@ -0,0 +1,220 @@
+import json
+import unittest.mock
+
+from utils.testing.base import ExtendedGraphQLTestCase
+from utils.testing.factories.ecommerce import OrderFactory, ProductFactory
+from utils.testing.factories.organizations import OrganizationFactory
+from utils.testing.factories.users import IndokUserFactory
+
+from apps.ecommerce.models import Product
+from apps.ecommerce.mutations import AttemptCapturePayment, InitiateOrder
+
+
+class EcommerceBaseTestCase(ExtendedGraphQLTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.indok_user = IndokUserFactory()
+        self.organization = OrganizationFactory()
+        self.product_1 = ProductFactory()
+        self.product_2 = ProductFactory()
+        self.order = OrderFactory(product=self.product_1, user=self.indok_user)
+
+
+class EcommerceResolversTestCase(EcommerceBaseTestCase):
+    """
+    Testing all resolvers for ecommerce
+    """
+
+    def test_resolve_products(self):
+        query = """
+                query products {
+                    products {
+                        id
+                        name
+                        description
+                        price
+                    }
+                }
+                """
+        response = self.query(query)
+        self.assertResponseNoErrors(response)
+
+        # Fetching content of response
+        content = json.loads(response.content)
+
+        # There are two products in the database
+        self.assertEqual(len(content["data"]["products"]), 2)
+
+    def test_resolve_order(self):
```

Comment:
Could consider splitting this into two separate tests, one for authorized users, and one for unauthorized users.